src/blog/2019-02-28-learning-with-dockter/index.md

```diff
@@ -0,0 +1,31 @@
+---
+extends: blog/_post.html
+title: Learning Docker through Dockter
+author: Aleksandra Pawlik
+date: 28 February 2019
+abstract: |
+    This is a short summary of my, somewhat painful, route to discovery that what I thought I knew about Docker was in fact just the tip of the iceberg. Fortunately to me, working in Stencila I upgraded my knowledge through testing and using Dockter, a container image builder for researchers.
+forum_topic:
+image: docker-error.png
+---
+
+This is a short summary of my, somewhat painful, route to discovery that what I thought I knew about Docker was in fact just the tip of the iceberg. Fortunately to me, working in Stencila I upgraded my knowledge through testing and using [Dockter](https://stencila.github.io/dockter/),  a container image builder for researchers.
+
+I used Docker for the first time several years ago when it entered the world of computational research. Working with computational research communities I was well aware of the problems with reproducibility due to dependency hell. Docker images appeared to be (and are!) a great solution. However, not actually doing
+computational research myself, I only used Docker a couple of times following a tutorial to build an image just to see how it would go. It went all fine, as usually well prepared tutorials present the user with a typical and uncomplicated case. I have run several Docker images that I got from the depot. That has also worked fine for me. And that would be it when it came to my hands-on experience with Docker. 
```

Comment:
Might be nice to use the common Docker terminology:
"from the depot" -> "from the registry" or "from the hub"